formats/cbor/commonTest/src/kotlinx/serialization/cbor/SampleClasses.kt

```diff
@@ -31,7 +31,7 @@ data class NumberTypesUmbrella(
 )
 
 @Serializable
-data class BinaryPayload(
-    val foo: ByteString,
-    val bar: ByteString
+data class ByteArrayUmbrella(
+    @ByteString val foo: ByteArray,
+    @ByteString val bar: ByteArray
```

Comment:
Would it make more sense to have one field of each encoding, instead of both byte strings?

Reply:
Thanks for catching this. I intended to have one of each and switched to both be `@ByteString` because the assertion payload had them as both byte string (and I was trying to make the test pass). I'll fix this.